docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1517,6 +1517,93 @@ rpc-http-port="3435"
 Specifies HTTP JSON-RPC listening port (TCP).
 The default is 8545. Ports must be [exposed appropriately](../../HowTo/Find-and-Connect/Configuring-Ports.md). 
 
+### rpc-http-tls-enabled
+
+```bash tab="Syntax"
+--rpc-http-tls-enabled
+```
+
+```bash tab="Environement Variable"
+BESU_RPC_HTTP_TLS_ENABLED=true
```

Comment:
I need to check this, have you test that this env variable works as expected?

Reply:
@usmansaleem, no I have not tested this. Will give it try when I test this feature again.